### ISSUE TITLE
Uncaught TypeError: Cannot read property 'scrollTop' of undefined

### ISSUE THREAD

**Issue by Arcanemagus** [jesseweed/seti-ui#398]
[Enter steps to reproduce:]

Unfortunately I haven't found reliable steps to reproduce, when this happens it is always on loading a _second_ Atom window (right click -> open with Atom) when you already have one open... but it's not 100% every time you load a secondary window.

**Atom**: 1.16.0-beta0 x64
**Electron**: 1.3.13
**OS**: Microsoft Windows 10 Enterprise
**Thrown From**: [seti-ui](https://github.com/jesseweed/seti-ui) package 1.6.1

### Stack Trace

Uncaught TypeError: Cannot read property 'scrollTop' of undefined

```
At /C:/Users/Landon%20Abney/.atom/packages/seti-ui/lib/headers.coffee:9

TypeError: Cannot read property 'scrollTop' of undefined
    at updateTreeViewHeaderPosition (/packages/seti-ui/lib/headers.coffee:8:43)
    at /packages/seti-ui/lib/headers.coffee:29:5
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at Project.module.exports.Project.addPath (/app.asar/src/project.js:288:29)
    at AtomEnvironment.module.exports.AtomEnvironment.openLocations (/app.asar/src/atom-environment.js:1195:26)
    at EventEmitter.outerCallback (/app.asar/src/application-delegate.js:229:18)
    at emitThree (events.js:121:20)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
```

### Non-Core Packages

```
atom-autocomplete-php 0.22.0 
atom-beautify 0.29.17 
atom-jade 0.3.0 
atom-typescript 10.1.15 
autocomplete-python 1.8.60 
busy-signal 1.3.0 
docblockr 0.9.1 
editorconfig 2.2.2 
file-watcher 1.1.0 
flow-ide 1.3.0 
highlight-selected 0.12.0 
intentions 1.1.2 
language-babel 2.56.2 
language-chef 0.9.0 
language-cjson 0.0.1 
language-docker 1.1.7 
language-elixir 0.17.0 
language-fortran 2.1.0 
language-glsl 2.0.1 
language-haml 0.24.1 
language-haskell 1.12.1 
language-ini 1.18.0 
language-jade 0.7.2 
language-jolie 0.4.5 
language-kotlin 0.5.0 
language-moonscript 1.7.1 
language-postcss 1.3.0 
language-powershell 4.0.0 
language-puppet 0.22.0 
language-r 0.4.1 
language-reg 0.0.0 
language-swift 0.5.0 
language-vue 0.21.2 
linter 1.99.0 
linter-alex 3.2.0 
linter-coffeelint 1.2.1 
linter-csslint 1.3.4 
linter-eslint 8.1.3 
linter-flake8 2.1.4 
linter-htmlhint 1.3.2 
linter-js-yaml 1.2.7 
linter-jsonlint 1.3.0 
linter-kotlin 0.2.0 
linter-markdown 3.1.0 
linter-perl 0.8.0 
linter-perlcritic 1.4.0 
linter-php 1.3.1 
linter-phpcs 1.5.10 
linter-phpmd 1.5.3 
linter-proselint 3.2.1 
linter-pylama 0.8.13 
linter-pylint 1.2.2 
linter-rails-best-practices 0.2.2 
linter-reek 2.2.0 
linter-rubocop 1.1.0 
linter-ruby 1.2.5 
linter-stylelint 3.9.0 
linter-tidy 2.3.0 
linter-ui-default 0.0.3 
mapfile-grammar 0.1.0 
merge-conflicts 1.4.4 
minimap 4.26.8 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.5.0 
minimap-linter 1.6.0 
minimap-selection 4.4.0 
pigments 0.39.0 
sequential-number 0.5.0 
seti-ui 1.6.1 
sort-lines 0.14.0 
Stylus 3.1.0 
svn 0.0.13 
tab-control 0.6.9 
toggle-quotes 1.0.1 
trailing-spaces 0.4.0 
```

**Comment by timdp** [comment]
Also happens on Linux. Likely an issue with the Atom beta.

**Comment by rcrooks** [comment]
Also happens when adding or removing a project folder in the tree view

**Comment by Arcanemagus** [comment]
Updated to that version, will let you know if I see it again.

**Comment by Alhadis** [comment]
In the meantime, I think you can close this, since this issue refers to a bug [I fixed recently](https://github.com/jesseweed/seti-ui/pull/390). ;)

**Comment by Arcanemagus** [comment]
Done 😉

**Comment by mgween** [comment]
I am getting this error any time I add or remove a project folder.

**Comment by Arcanemagus** [comment]
@mgween Please update your version of `seti-ui`, this should be fixed in version v1.7.0 and up.

**Comment by mgween** [comment]
@Arcanemagus I cannot update. When I check for available updates, I get `undefined`. Same problem as described [here](https://discuss.atom.io/t/solved-fetching-outdated-packages-and-themes-failed/39539).

**Comment by Arcanemagus** [comment]
One of the other things fixed in v1.7.0 was how it broke APM 😛. If you re-install it things should all be working.
